app/src/main/java/org/simple/clinic/shortcodesearchresult/ShortCodeSearchResultScreen.kt

```diff
@@ -64,7 +67,15 @@ class ShortCodeSearchResultScreen(context: Context, attributes: AttributeSet) :
 
   private val adapter = ItemAdapter(SearchResultsItemType.DiffCallback())
 
-  private lateinit var disposable: Disposable
+  private val events by unsafeLazy {
+    Observable
+        .merge(
+            searchPatientClicks(),
+            patientItemClicks()
+        )
+        .compose(ReportAnalyticsEvents())
```

Comment:
Doesn't the controller already have reporting at this point?

Reply:
There is no controller in this intermediate architecture. We never ended up reporting these events to analytics at all, looks like.

Reply:
Ah okay 👍🏽